diff --git b/py-issuer/config/app_config/config_countries.py a/py-issuer/config/app_config/config_countries.py
index 9e0feed..d47254d 100644
--- b/py-issuer/config/app_config/config_countries.py
+++ a/py-issuer/config/app_config/config_countries.py
@@ -46,7 +46,8 @@ class ConfCountries:
                 "eu.europa.ec.eudi.over18_mdoc",
                 "eu.europa.ec.eudi.loyalty_mdoc",
                 "eu.europa.ec.eudi.pseudonym_over18_mdoc",
-                "eu.europa.ec.eudi.pseudonym_over18_mdoc_deferred_endpoint"
+                "eu.europa.ec.eudi.pseudonym_over18_mdoc_deferred_endpoint",
+                "eu.europa.ec.eudi.nl.pensioenoverzicht",
             ],
             "dynamic_R2": cfgserv.service_url + "dynamic/form_R2",
         },
