diff --git a/app/route_oid4vp.py b/app/route_oid4vp.py
index b3a5068..ea9104f 100644
--- a/app/route_oid4vp.py
+++ b/app/route_oid4vp.py
@@ -45,7 +45,8 @@ from app.data_management import oid4vp_requests, form_dynamic_data
 @oid4vp.route("/oid4vp", methods=["GET"])
 def openid4vp():
 
-    url = "https://dev.verifier-backend.eudiw.dev/ui/presentations"
+    #url = "https://dev.verifier-backend.eudiw.dev/ui/presentations"
+    url = cfgservice.verifier_backend_url + "/ui/presentations"
     payload_cross_device = json.dumps(
         {
             "type": "vp_token",
@@ -194,14 +195,16 @@ def openid4vp():
     oid4vp_requests.update({id:{"response": response_same, "expires":datetime.now() + timedelta(minutes=cfgservice.deffered_expiry)}})
 
     deeplink_url = (
-        "eudi-openid4vp://dev.verifier-backend.eudiw.dev?client_id="
+        #"eudi-openid4vp://dev.verifier-backend.eudiw.dev?client_id="
+        "eudi-openid4vp://" + cfgservice.verifier_backend_domain + "/?client_id="
         + response_same["client_id"]
         + "&request_uri="
         + response_same["request_uri"]
     )
 
     qr_code_url = (
-        "eudi-openid4vp://dev.verifier-backend.eudiw.dev?client_id="
+        #"eudi-openid4vp://dev.verifier-backend.eudiw.dev?client_id="
+        "eudi-openid4vp://" + cfgservice.verifier_backend_domain + "/?client_id="
         + response_cross["client_id"]
         + "&request_uri="
         + response_cross["request_uri"]
@@ -244,7 +247,8 @@ def getpidoid4vp():
         response_code = request.args.get("response_code")
         presentation_id = oid4vp_requests[request.args.get("session_id")]["response"]["presentation_id"]
         url = (
-            "https://dev.verifier-backend.eudiw.dev/ui/presentations/"
+            #"https://dev.verifier-backend.eudiw.dev/ui/presentations/"
+            cfgservice.verifier_backend_url + "/ui/presentations/"
             + presentation_id
             + "?nonce=hiCV7lZi5qAeCy7NFzUWSR4iCfSmRb99HfIvCkPaCLc="
             + "&response_code=" + response_code
@@ -254,7 +258,8 @@ def getpidoid4vp():
         presentation_id = request.args.get("presentation_id")
 
         url = (
-            "https://dev.verifier-backend.eudiw.dev/ui/presentations/"
+            #"https://dev.verifier-backend.eudiw.dev/ui/presentations/"
+            cfgservice.verifier_backend_url + "/ui/presentations/" + 
             + presentation_id
             + "?nonce=hiCV7lZi5qAeCy7NFzUWSR4iCfSmRb99HfIvCkPaCLc="
         )
@@ -321,6 +326,7 @@ def getpidoid4vp():
 
     attributesForm.update({"issuing_country": "FC"})
     attributesForm.update({"issuing_authority": doctype_config["issuing_authority"]})
+    attributesForm.update({"timestamp": datetime.timestamp(datetime.now())})
 
     user_id = generate_unique_id()
     form_dynamic_data[user_id] = attributesForm
@@ -344,4 +350,4 @@ def getpidoid4vp():
         attributes=presentation_data,
         user_id="FC." + user_id,
         redirect_url=cfgservice.service_url + "dynamic/redirect_wallet",
-    )
\ No newline at end of file
+    )
