diff --git a/app/route_oidc.py b/app/route_oidc.py
index f3e194a..7cd38d0 100644
--- a/app/route_oidc.py
+++ b/app/route_oidc.py
@@ -446,7 +446,8 @@ def pid_authorization_get():
     presentation_id = request.args.get("presentation_id")
 
     url = (
-        "https://dev.verifier-backend.eudiw.dev/ui/presentations/"
+#        "https://dev.verifier-backend.eudiw.dev/ui/presentations/"
+        cfgservice.verifier_backend_url + "/ui/presentations/"
         + presentation_id
         + "?nonce=hiCV7lZi5qAeCy7NFzUWSR4iCfSmRb99HfIvCkPaCLc="
     )
@@ -1142,4 +1143,4 @@ def service_endpoint(endpoint):
 
 @oidc.errorhandler(werkzeug.exceptions.BadRequest)
 def handle_bad_request(e):
-    return "bad request!", 400
\ No newline at end of file
+    return "bad request!", 400
